### PR TITLE
Status/2024Q3: Add simd.adoc

### PR DESCRIPTION
**Pull request by soppelmann**
Create AsciiDoc report for libc simd improvements made within 2024Q3 as part of Google Summer Of Code (GSoC)

### CODE REVIEW

File: website/content/en/status/report-2024-07-2024-09/simd.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Links: +
+link:https://www.youtube.com/live/OzX38cWdivc?si=VsMrEmT_IdKpjv7W&t=22070[EuroBSDCon 2024 presentation] URL: link:https://www.youtube.com/live/OzX38cWdivc?si=VsMrEmT_IdKpjv7W&t=22070[] +
+link:https://summerofcode.withgoogle.com/programs/2024/projects/TKRS35FA[Google Summer of Code (GSoC) project description] URL: link:https://summerofcode.withgoogle.com/programs/2024/projects/TKRS35FA[] +
+link:https://man.freebsd.org/cgi/man.cgi?query=simd&sektion=7&manpath=FreeBSD+15.0-CURRENT[simd(7)] URL: link:https://man.freebsd.org/cgi/man.cgi?query=simd&sektion=7&manpath=FreeBSD+15.0-CURRENT[] +
+
+Contact: Getz Mikalsen <getz@FreeBSD.org>
+
+The porting effort of the SIMD enhanced libc string functions from amd64 to
+aarch64 has been successfully completed. There now exists optimized
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/simd.adoc:11]: s/exists/are/ (number agreement + simplify vocabulary)

File: website/content/en/status/report-2024-07-2024-09/simd.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+implementations already available as part of the ARM optimized subroutines
+library.
+There is also a presentation regarding the general method for these
+methods from EuroBSDCon 2024 available on YouTube with a short description in
+the end of how the porting has been done with regards to the aarch64 architecture.
+
+These enhancements significantly improve performance of string functions for
+all FreeBSD systems on the aarch64 platform.
+
+The code is currently undergoing acceptence testing in the form of an exp-run
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/simd.adoc:22]: s/acceptence/acceptance/

File: website/content/en/status/report-2024-07-2024-09/simd.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
+=== SIMD enhancements for aarch64
+
+Links: +
+link:https://www.youtube.com/live/OzX38cWdivc?si=VsMrEmT_IdKpjv7W&t=22070[EuroBSDCon 2024 presentation] URL: link:https://www.youtube.com/live/OzX38cWdivc?si=VsMrEmT_IdKpjv7W&t=22070[] +
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/simd.adoc:4]: In addition to that, could you provide a link to the slides or presentation notes, for accessibility?

**Review by pauamma — APPROVED** [review]

**Review by lsalvadore — APPROVED** [review]
Approved, but it needs to have one sentence per line. Fixable at merge time.

**Comment by lsalvadore** [comment]
Merged, thanks!

**Pull request closed without merging**